cmd/enroll.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+)
+
+type enrollTokenData struct {
+	Token string `json:"token"`
+}
+
+type enrollResponse struct {
+	Success bool            `json:"success"`
+	Message string          `json:"message"`
+	Data    enrollTokenData `json:"data"`
+}
+
+func getApiUrl(firstLetter string) (*string, error) {
+	apiUrls := map[string]string{
+		"P": "https://api.shopmonkey.cloud/",
+		"S": "https://sandbox-api.shopmonkey.cloud/",
+		"E": "https://edge-api.shopmonkey.cloud/",
+		"L": "http://localhost:3101",
+	}
+
+	if url, exists := apiUrls[firstLetter]; exists {
+		return &url, nil
+	}
+	return nil, errors.New("invalid first letter")
```

Comment:
maybe make it a bit more vague
```suggestion
	return nil, errors.New("invalid code")
```